src/components/Nav.js

```diff
@@ -1,23 +1,61 @@
-import React from "react";
+import React, { useState } from "react";
 import {
   Drawer,
-  Divider,
   List,
   ListItem,
+  ListItemIcon
 } from "@material-ui/core";
 import "./Nav.scss";
-import Tooltip from '@material-ui/core/Tooltip';
-import HomeIcon from '@material-ui/icons/Home';
+import { withStyles } from '@material-ui/styles';
 import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 import AddBoxIcon from '@material-ui/icons/AddBox';
 import { useHistory, withRouter } from "react-router-dom";
 import { Auth } from "aws-amplify";
 import { connect } from 'react-redux'
 import { logout } from '../actions/UserActions'
+import Logo from '../pages/member/images/biztech.svg'
+
+const styles = {
+  list: {
+    width: '170px',
+    paddingTop: '95px'
+  },
+  icon: {
+    width: '49px',
+    paddingTop: '28px',
+    paddingBottom: '28px',
+    cursor: 'pointer',
+    paddingLeft: '54px'
+  },
+  iconSize: {
+    fontSize: '30'
+  },
+  logout: {
+    marginTop: '330px',
+    cursor: 'pointer',
+    paddingLeft: '48px',
+    color: '#FFFFFF'
+  },
+  paper: {
+    backgroundColor: '#070F21'
+  }
+}
 
 function Nav(props) {
+  const { classes } = props
   const history = useHistory();
 
+  const selected = { color: '#7AD040', fontSize: '30', width: '43px' }
+  const unselected = { color: '#FFFFFF', fontSize: '30', width: '43px' }
+  const barSelected = { borderLeft: '6px solid #7AD040', height: '45px', marginTop: '28px', paddingBottom: '28px', backgroundPositionX: 'left' }
+  const barUnselected = { borderLeft: '6px solid #070F21', height: '45px', marginTop: '28px', paddingBottom: '28px' }
+
+  const pathname = window.location.pathname
+  const [homeBar, setHomeBar] = useState(pathname === '/' ? barSelected : barUnselected)
+  const [eventsBar, setEventsBar] = useState(pathname === '/event/new' ? barSelected : barUnselected)
+  const [home, setHome] = useState(pathname === '/' ? selected : unselected)
+  const [events, setEvents] = useState(pathname === '/event/new' ? selected : unselected)
```

Comment:
could we maybe reduce these to one 'selected' item
```
const [selectedItem, setSelectedItem] ...
```

Reply:
thats smart bro